soc/xtensa/intel_adsp/ace/include/intel_ace15_mtpm/adsp_shim.h

```diff
@@ -6,50 +6,68 @@
 #ifndef ZEPHYR_SOC_INTEL_ADSP_SHIM_H_
 #define ZEPHYR_SOC_INTEL_ADSP_SHIM_H_
 
-/* The "shim" block contains most of the general system control
- * registers on cAVS platforms.  While the base address changes, it
- * has remained largely, but not perfectly, compatible between
- * versions.
+#ifndef _ASMLANGUAGE
+
+/**
+ * DfPMCCU
+ * Power Management / Clock Control (HST) Registers
+ *
+ * Offset: 0x00071AC0h + 4000000h * f //f = 0 - HFC - 1
+ * These registers block (HST domain) are for general power management
+ * and clock control operation for DSP FW.
  */
+struct ace_dfpmcch {
+	uint32_t dfspsreq;	/* Offset: 0xc0 */
```

Comment:
Wrong offsets. `dfspsreq` addr is 71ac0, so the comment should be `Offset: 0x00`. They are ok in `ace_dfpmccu` though

Reply:
@kfrydryx please re-review and resolve the conversation if it is o.k for you ;)